hack/components/yaml-utils.sh

```diff
@@ -2,8 +2,12 @@
 
 set -xeo pipefail
 
+source hack/components/docker-utils.sh
+
+export OCI_BIN=${OCI_BIN:-$(docker-utils::determine_cri_bin)}
+
 function __yq() {
-  docker run --rm -i -v ${PWD}:/workdir:Z mikefarah/yq:3.3.4 yq "$@"
+  ${OCI_BIN} run --rm -v ${PWD}:/workdir:Z docker.io/mikefarah/yq:3.3.4 yq "$@"
```

Comment:
Removed the `-i` because it is not needed here, and with it we might see this flake:

https://bugzilla.redhat.com/show_bug.cgi?id=1754922
```
++ podman run --rm -i -v /tmp/cnao/cluster-network-addons-operator:/workdir:Z docker.io/mikefarah/yq:3.3.4 yq r components.yaml 'components."macvtap-cni".url'
Error: error attaching to container 4fb46303703711405a6006f2ec6cce58c77c09c3bbcb7e1439b82b7a8e69d2c0: read unixpacket @->/proc/self/fd/14/attach: read: connection reset by peer
+ MACVTAP_URL=https://github.com/kubevirt/macvtap-cni
make: *** [Makefile:201: bump-macvtap-cni] Error 126
```